File: app/src/main/java/com/example/cykashoppinglist/mapper/GenericMapper.java
package com.example.cykashoppinglist.mapper;

import com.example.cykashoppinglist.entity.GenericItem;

import org.json.JSONException;
import org.json.JSONObject;

public class GenericMapper {
	public static JSONObject toJson(GenericItem item) {
		try {
			JSONObject jsonObject = new JSONObject();
			jsonObject.put("content", item.getName());
			return jsonObject;
		} catch (JSONException e) {
			e.printStackTrace();
		}
		return null;
	}
}

File: app/src/main/java/com/example/cykashoppinglist/mapper/GenericMapper.kt
package com.example.cykashoppinglist.mapper

import com.example.cykashoppinglist.entity.GenericItem
import org.json.JSONException
import org.json.JSONObject

object GenericMapper {
	fun toJson(item: GenericItem): JSONObject? {
		try {
			val jsonObject = JSONObject()
			jsonObject.put("content", item.name)
			return jsonObject
		} catch (e: JSONException) {
			e.printStackTrace()
		}
		return null
	}
}